.rubocop.yml

```diff
@@ -11,3 +11,5 @@ Style/FrozenStringLiteralComment:
   Exclude:
     - "**/bin/*"
     - "**/exe/*"
+    - "spec/**/*"
```

Comment:
Is there a specific reason for NOT enforcing this in specs? I like the consistency and the fact that it may increase the spec's performance even if just by a tiny bit, and I'm also a fan of not manipulating objects unless absolutely necessary.

Reply:
It's a trade of (as everything!), I consider the magic comment a cost that needs to be paid, and if there's no strong reason to pollute the file with this kind of noise it's better.

Being more specific, I think performance might actually be worse (or unaffected at most) for specs as there's a high number of single use strings coming from example descriptions. The point about string-mutability is more alluring but given how Ruby is designed in this case it's probably just a drop in the ocean.